artifactory/commands/generic/upload.go

```diff
@@ -174,6 +174,7 @@ func (uc *UploadCommand) upload() (err error) {
 			log.Error(err)
 		}
 	}
+	log.Info("Uploaded successfully!")
```

Comment:
I suggest removing this new log message, as it'll add one additional info message for each file upload. I'm not sure this is needed.